tests/simulator/__init__.py

```
@@ -7,7 +7,9 @@
 from datetime import datetime
 
 import pyphare.pharein as ph
+from pyphare.core.box import Box
 from pyphare.pharein import ElectronModel
+from pyphare.core.box import Box
```

Ruff (0.14.4)

Redefinition of unused `Box` from line 10

Remove definition: `Box`

(F811)